site/access-control.xml

```diff
@@ -328,7 +391,17 @@ limitations under the License.
             href="https://github.com/gotthardp/rabbitmq-auth-backend-ip-range">source
             IP range backend</a> for authorisation:
           </p>
-            
+
+<pre class="example">
+# rabbitmq.conf
+#
+auth_backends.1.authn = internal
+# uses module name because this backend is from a 3rd party
+auth_backends.1.authz = rabbit_auth_backend_ip_range
```

Comment:
[See this internals guide](https://github.com/rabbitmq/internals/blob/master/authorization_and_authentication.md).